packages/grafana-toolkit/src/config/webpack/loaders.ts

```diff
@@ -139,34 +104,21 @@ export const getStyleLoaders = () => {
 };
 
 export const getFileLoaders = () => {
-  const shouldExtractCss = hasThemeStylesheets();
-
   return [
     {
       test: /\.(png|jpe?g|gif|svg)$/,
-      use: [
-        shouldExtractCss
-          ? {
-              loader: require.resolve('file-loader'),
-              options: {
-                outputPath: '/',
-                name: '[path][name].[ext]',
-              },
-            }
-          : // When using single css import images are inlined as base64 URIs in the result bundle
-            {
-              loader: 'url-loader',
-            },
-      ],
+      type: 'asset/resource',
+      generator: {
+        publicPath: `img/`,
+        outputPath: 'img/',
+      },
     },
     {
       test: /\.(woff|woff2|eot|ttf|otf)(\?v=\d+\.\d+\.\d+)?$/,
-      loader: require.resolve('file-loader'),
-      options: {
-        // Keep publicPath relative for host.com/grafana/ deployments
-        publicPath: `public/plugins/${getPluginId()}/fonts`,
-        outputPath: 'fonts',
-        name: '[name].[ext]',
+      type: 'asset/resource',
+      generator: {
+        publicPath: `fonts/`,
```

Comment:
I am not sure why we had `public/plugins/.../fonts` here before and if there is anything we need to consider here @jackw

Reply:
@jackw and I looked at this together and using just this path generates the correct file output and refs.

Reply:
@tolzhabayev I don't know for sure but it seems like; there were some changes to the way these loaders worked out paths between `webpack@4` and `webpack@5`. Whilst testing with @academo it looks like these paths resolve themselves relatively which we believe will give the most consistent path resolution.